packages/backend-auth/src/factory.ts

```diff
@@ -45,7 +46,8 @@ export type AmplifyAuthProps = Expand<
 /**
  * Singleton factory for AmplifyAuth that can be used in Amplify project files
  */
-class AmplifyAuthFactory implements ConstructFactory<BackendAuth> {
+export class AmplifyAuthFactory implements ConstructFactory<BackendAuth> {
```

Comment:
Need to export this class to access call count for testing. 

Made sure not to export unnecessary code from `src/index.ts`: 
https://github.com/aws-amplify/amplify-backend/pull/1046/files#diff-a43553012e47a2b89d0a4aacfd2d9e7ec9358de0bafc35399392034ca5538500L1

Reply:
I would leave a note in the JS doc that this class is only exported for testing and should not be exported from the package